packages/katello/rubygem-pulp_file_client/rubygem-pulp_file_client.spec

```diff
@@ -0,0 +1,97 @@
+# Generated from pulp_file_client-0.0.1b10.dev.1557779852.gem by gem2rpm -*- rpm-spec -*-
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name pulp_file_client
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.1b10.dev.1557779852
+Release: 1%{?dist}
+Summary: Pulp 3 API Ruby Gem
+Group: Development/Languages
+License: GPLv2
+URL: https://openapi-generator.tech
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby >= 1.9
+Requires: %{?scl_prefix_ruby}ruby(rubygems) > 1.3.1
+Requires: %{?scl_prefix}rubygem(typhoeus) >= 1.0
+Requires: %{?scl_prefix}rubygem(typhoeus) < 2
+Requires: %{?scl_prefix}rubygem(typhoeus) >= 1.0.1
+Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1
+Requires: %{?scl_prefix_ruby}rubygem(json) < 3
+Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1.0
```

Comment:
Three json requires?

Reply:
its what gem2rpm produced, but it does look weird, i will adjust :)